.npmrc

```diff
@@ -5,5 +5,5 @@ enable-pre-post-scripts=true
 
 # node-linker=hoisted
 strict-peer-dependencies=false
-engine-strict=true
+# engine-strict=true
```

Comment:
I disabled this because, even after several reinstalls using different methods, pnpm fails to detect my node version, doesn't matter if i use `pnpm env use`, homebrew node, manual node install, it just fails. according to some issues in pnpm, this seems to be a long-standing bug.

```shell
❯ pnpm i
Scope: all 23 workspace projects
 ERR_PNPM_UNSUPPORTED_ENGINE  Unsupported environment (bad pnpm and/or Node.js version)

Your Node version is incompatible with "@babel/runtime@7.27.0".

Expected version: >=6.9.0
Got: false

This is happening because the package's manifest has an engines.node field specified.
To fix this issue, install the required Node version.
```